daffodil-lib/src/test/resources/test/validation/myData5.dat

```diff
@@ -0,0 +1 @@
+12:345678
```

Comment:
I just copied the files (not content) from another resource directory.  Could be something odd with them though.

Reply:
Looks like these other files are UTF-16. UTF-8/ascii is prefered unless directly testing for UTF-16.

Reply:
updated the encoding

Reply:
I noticed you also renamed a test .dat file:

daffodil-lib/src/test/resources/test/validation/myData5.dat → ...t/resources/test/validation/testData1.dat 

How is this file used?  I searched through all the GitHub diffs in my browser and I couldn't find any other occurrence of the filename "testData1.dat".

Reply:
it wasnt used. must have copied it over with the other files planning on use but didnt.

removed it

Reply:
renamed these files to prevent future confusion with the files in the other projects with the only difference being encoding... that could lead to a very bad day.